packages/ui-causal-graph-editor/src/shared/rendering/engine.tsx

```diff
@@ -227,6 +227,9 @@ export class Engine extends PIXI.utils.EventEmitter<EngineEvents> {
     /** whether zoom on scroll should be enabled only when the graph is focused */
     private requireFocusToZoom: boolean;
 
+    /** Whether the graph is currently focused */
+    private isFocused: boolean;
```

Comment:
The property should be initialized to `false` in the constructor, otherwise it would really be `boolean | undefined`